app/konnect/analytics/services-and-routes.md

```diff
@@ -32,7 +31,7 @@ see [custom reports](/konnect/analytics/generate-reports/).
 You can view traffic health and performance of an individual service, including across all of a
 service's versions.
 
-From the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
+From the {% konnect_icon api-product %} [**API Products builder**](https://cloud.konghq.com/servicehub), select a service.
```

Comment:
Are we still using the term "service"?  I thought we were supposed to say API product.